detox/test/e2e/12.animations.test.js

```diff
@@ -51,7 +51,7 @@ describe('Animations', () => {
       await expect(element(by.id('UniqueId_AnimationsScreen_afterAnimationText'))).toNotExist();
     });
 
-    it(`should wait during delays shorter than 1.5s (driver: ${driver})`, async () => {
+    it(`:ios: should wait during delays shorter than 1.5s (driver: ${driver})`, async () => {
```

Comment:
Why did you disable tests for Android?

Reply:
as I said in [this comment](https://github.com/wix/Detox/pull/1104#issuecomment-452407831) these tests look flaky? cause looks like coming from [here](https://github.com/wix/Detox/pull/1104#issuecomment-452413650)

Reply:
I'll try to run them from updated master and let you know the result